packages/lib-classifier/src/plugins/tasks/SurveyTask/components/components/Choice/components/ConfusedWith/ConfusedWith.js

```diff
@@ -0,0 +1,100 @@
+import { Box, DropButton } from 'grommet'
+import PropTypes from 'prop-types'
+import React from 'react'
+import styled from 'styled-components'
+import { SpacedHeading } from '@zooniverse/react-components'
+
+import Confusion from './components/Confusion'
+
+import counterpart from 'counterpart'
+import en from './locales/en'
+
+counterpart.registerTranslations('en', en)
+
+export const StyledDropButton = styled(DropButton)`
+  background-color: ${props => props.open
+    ? props.theme.global.colors['accent-2']
+    : props.theme.dark
+      ? props.theme.global.colors['dark-3']
+      : props.theme.global.colors['neutral-6']
```

Comment:
Nested ternaries can be a bit hard to read, so I'd suggest that this be a function in the component to determine the color and pass down a `backgroundColor` prop to use here?